### PR TITLE
Create UNCW_Dogan.yaml

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

**Comment by brianhlin** [comment]
@opensciencegrid/project-office do you all approve of the new institution, `University of North Carolina Wilmington`?

**Comment by jstathas** [comment]
Approved

**Comment by brianhlin** [comment]
@jstathas great! Let us know what the OSG ID ends up being so that we can add it here and close the loop

**Comment by jstathas** [comment]
The OSG ID is 
https://osg-htc.org/iid/14m1189m446l

File: projects/UNCW_Dogan.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Department: Computer Science
+Description: Medical Question Answering https://medvidqa.github.io
+FieldOfScience: Computer Science
+FieldOfScienceID: '11.0102'
+InstitutionID: Unknown
```
Comment (brianhlin) [inline projects/UNCW_Dogan.yaml:5]:
```suggestion
InstitutionID: https://osg-htc.org/iid/14m1189m446l
```

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**